src/eprunner.py

```diff
@@ -49,6 +50,14 @@ def run_simulation(self):
             precess run object, which contains all stdout/stderr, will be saved by the save_log method
 
         """
+
+        # copy weather file to the simulation output folder to allow simulations in parallel
+        Path(self.output_path).mkdir(parents=True, exist_ok=True)
+        shutil.copyfile(
```

Comment:
This is a very trivial comment. Could you see if we could use ```.join()``` method for the string concatenation as it is more efficient ([LINK](https://stackoverflow.com/questions/39312099/why-is-join-faster-than-in-python))?